firefox-ios/firefox-ios-tests/Tests/ClientTests/OnboardingTests/OnboardingServiceTests.swift

```diff
@@ -324,18 +324,14 @@ final class OnboardingServiceTests: XCTestCase {
     func testHandleAction_SyncSignIn_UpdatesActivityEventHelper() {
         // Given
         let activityEventHelper = MockActivityEventHelper()
-        let expectation = XCTestExpectation(description: "Completion called")
```

Comment:
can you confirm that after removing this expectation that if we run this test multiple times (i.e. `2000`), it doesn't fail.